<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget_Options</class>
 <widget class="QWidget" name="Widget_Options">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>1</y>
     <width>585</width>
     <height>408</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="assets.qrc">:/icons/menu_on.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="MyLabelAnimated" name="l_hide_menu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="assets.qrc">:/icons/flecha_izq.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="widget_buttons" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>241</width>
     <height>341</height>
    </rect>
   </property>
   <widget class="MyThreeStatesLabelAnimated" name="l_config_alarms">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>178</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="assets.qrc">:/icons/config_alarms_off.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="MyThreeStatesLabelAnimated" name="l_config_fecha_hora">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>209</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="assets.qrc">:/icons/config_fecha_hora_off.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="MyThreeStatesLabelAnimated" name="l_download_data">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>174</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="assets.qrc">:/icons/download_data_off.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="MyThreeStatesLabelAnimated" name="l_pacient_data">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>167</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="assets.qrc">:/icons/pacient_data_off.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="MyThreeStatesLabelAnimated" name="l_log_information">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>260</y>
      <width>197</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="assets.qrc">:/icons/log_information_off.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="MyThreeStatesLabelAnimated" name="l_otros_ajustes">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>139</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="assets.qrc">:/icons/otros_ajustes_off.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="MyThreeStatesLabelAnimated" name="l_calibracion_oxigeno">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>310</y>
      <width>226</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="assets.qrc">:/icons/calibracion_oxigeno_off.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pb_power_off">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>500</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Power off</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>widget_buttons</zorder>
  <zorder>l_hide_menu</zorder>
  <zorder>pb_power_off</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyLabelAnimated</class>
   <extends>QLabel</extends>
   <header location="global">mylabelanimated.h</header>
  </customwidget>
  <customwidget>
   <class>MyThreeStatesLabelAnimated</class>
   <extends>QLabel</extends>
   <header location="global">mythreestateslabelanimated.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="assets.qrc"/>
 </resources>
 <connections/>
</ui>
